cmd/controller-manager/main.go

```diff
@@ -63,34 +68,41 @@ func main() {
 	var itsName string
 	var wdsName string
 	var allowedGroupsString string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The [host]:port from which /metrics is served.")
-	flag.StringVar(&pprofAddr, "pprof-bind-address", ":8082", "The [host]:port fron which /debug/pprof is served.")
-	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.StringVar(&itsName, "its-name", "", "name of the Inventory and Transport Space to connect to (empty string means to use the only one)")
-	flag.StringVar(&wdsName, "wds-name", "", "name of the workload description space to connect to")
-	flag.StringVar(&allowedGroupsString, "api-groups", "", "list of allowed api groups, comma separated. Empty string means all API groups are allowed")
-	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+	var controllers []string
+	pflag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The [host]:port from which /metrics is served.")
+	pflag.StringVar(&pprofAddr, "pprof-bind-address", ":8082", "The [host]:port fron which /debug/pprof is served.")
+	pflag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	pflag.StringVar(&itsName, "its-name", "", "name of the Inventory and Transport Space to connect to (empty string means to use the only one)")
+	pflag.StringVar(&wdsName, "wds-name", "", "name of the workload description space to connect to")
+	pflag.StringVar(&allowedGroupsString, "api-groups", "", "list of allowed api groups, comma separated. Empty string means all API groups are allowed")
+	pflag.StringSliceVar(&controllers, "controllers", []string{}, "list of controllers to be started by the controller manager, comma separated, e.g. 'binding,status'")
```

Comment:
Since the meaning of the empty array is special, I think it should be mentioned in the doc string.

Reply:
Improved the doc string as suggested.